### PR TITLE
Fix backslashed replacements

### PR DESCRIPTION
**Pull request by pereirinha**
- Add support for backslashed URLs
- Fixes an issue with the rest requests

### CODE REVIEW

File: php/class-delivery.php [hunk 1]
```diff
@@ -897,8 +897,13 @@ public function convert_tags( $content, $context = 'view' ) {
 			$base                   = $type . ':' . $url;
 			$public_id              = ! is_admin() ? $relation['public_id'] . '.' . $relation['format'] : null;
 			$cloudinary_url         = $this->media->cloudinary_url( $relation['post_id'], array(), $relation['transformations'], $public_id );
-			$aliases[ $base . '?' ] = $cloudinary_url . '&';
-			$aliases[ $base ]       = $cloudinary_url;
+			if ( ! empty( $relation['slashed'] ) && $relation['slashed'] ) {
+				$aliases[ $base . '?' ] = trim( wp_json_encode( $cloudinary_url . '&' ), '"' );
+				$aliases[ $base ]       = trim( wp_json_encode( $cloudinary_url ), '"' );
```
Comment (DavidCramer) [inline php/class-delivery.php:902]: @pereirinha - Would be cleaner to do:
```suggestion
				$aliases[ $base . '?' ] = addcslashes( $cloudinary_url . '&', '/' );
				$aliases[ $base ]       = addcslashes( $cloudinary_url ), '/' );
```

File: php/class-delivery.php [hunk 2]
```diff
@@ -1433,8 +1438,12 @@ protected function set_usability( $item, $auto_sync = null ) {
 		$this->known[ $item['public_id'] ] = $item;
 		$scaled                            = self::make_scaled_url( $item['sized_url'] );
 		$descaled                          = self::descaled_url( $item['sized_url'] );
+		$scaled_slashed                    = trim( wp_json_encode( $scaled ), '"' );
+		$descaled_slashed                  = trim( wp_json_encode( $descaled ), '"' );
```
Comment (DavidCramer) [inline php/class-delivery.php:1442]: @pereirinha - Same as above.

Comment (DavidCramer) [inline php/class-delivery.php:1442]: @pereirinha - Same as above.

**Review by DavidCramer — CHANGES_REQUESTED** [review]
@pereirinha - added comments on the function I mentioned. didn't remember it then.

**Comment by pereirinha** [comment]
thanks @DavidCramer 

Updates are in, back to you ;)

**Review by DavidCramer — APPROVED** [review]

**Pull request merged**